Handbook/Resources.md

```diff
@@ -43,6 +43,7 @@ Useful materials that give an overview of Cook County's property tax system and
 - [:exclamation: The Cook County Property Tax Extension Process: A Primer](https://www.civicfed.org/civic-federation/publications/cook-county-property-tax-extension-process-primer-levies-tax-caps-and-) - Incredibly useful guide to the intricacies of the entire Cook County tax system
 - [:star: Why Are My Property Taxes So High?](https://www.chicagobusiness.com/crains-forum-taxes/forum-why-are-my-property-taxes-so-high) - Neat interactive from Crain's that compares two very different homes that pay roughly the same amount of property tax
 - [:star: How Lower-Income Americans Get Cheated on Property Taxes - NYTimes](https://www.nytimes.com/2021/04/03/opinion/sunday/property-taxes-housing-assessment-inequality.html) - Concise and accessible overview of the nation-wide issue of inequitable assessments
+- [:star: Chicago Inside Out - Places Journal](https://placesjournal.org/article/chicago-inside-out) - Deep dive into the roles and recent histories of the offices that comprise Cook County government, with a particular focus on the impact of property taxes on county services
```

Comment:
This article does a great job making salient some key confusions (like the fact that even though Cook County sends and collects bills, most taxpayer dollars fund non-county agencies, like city or school districts). Let's please retitle this to Chicago Inside Out (2018) - Places Journal, remove the star, and move it lower down either above or below the Chicago Triad Assessment Report (2018).